generate_report.py

```diff
@@ -40,36 +40,38 @@ def SVGFile(data: np.ndarray, data_max: np.float64, data_min: np.float64,
 
   if field == "scl_show":
     svgfile += (
-        f"<div id='{field}'><div class='column_left margin'>SCL capture</div>"
-        "<div class='column_right margin'>"
-        f"<svg viewBox='0 0 {width} {height * 1.2}' height={height * 0.7} width=auto>"
+        f"<div id='{field}'><div class='column_left'>SCL capture</div>"
+        "<div class='column_right'>"
+        f"<svg viewBox='0 0 {width} {height * 1.2 + 120}' style='height={height + 120};width=auto'>"
```

Comment:
Isn't this `style='height: {height + 120}; width: auto'` ? (use colon instead of equal sign)